test_common/harness/kernelHelpers.c

```diff
@@ -391,6 +395,12 @@ static int get_offline_compiler_output(std::ifstream &ifs,
             return -1;
         }
 
+        if(gExecutionMode == kGenerate) {
+            // in this mode we don't emit the compiler
+            // we fake a compilation error instead
+            return -1;
```

Comment:
If we're going to keep that approach, I think it's worth introducing a symbolic name for this error code.

Reply:
sure, could you point me to where the right place would be to add this symbolic name?